server/schedule/checker/checker_controller.go

```diff
@@ -76,6 +76,13 @@ func (c *Controller) CheckRegion(region *core.RegionInfo) []*operator.Operator {
 	// Don't check isRaftLearnerEnabled cause it maybe disable learner feature but there are still some learners to promote.
 	opController := c.opController
 
+	if cl, ok := c.cluster.(interface{ GetRegionLabeler() *labeler.RegionLabeler }); ok {
```

Comment:
Should we ignore **schedule deny label** for join state and replica checker? Dealing with unhealthy region does not make sense in most case.

Reply:
Yes,  `ignore from join state` is done.  But we do not ignore the label for `replica check` because `the replica operators generated by manual` can still work when `schedule=deny`.